python/test/test_cpp_algos.py

```diff
@@ -153,6 +155,27 @@ def test_sssp(graph: Graph):
     verify_sssp(graph, start_node, property_name)
 
 
+def test_ksssp(graph: Graph):
+    weight_name = "workFrom"
+    start_node = 0
+    report_node = 10
+    num_paths = 5
+
+    table = ksssp(graph, weight_name, start_node, report_node, num_paths)
+    stats = KssspStatistics(graph, weight_name, table, report_node)
+    paths = stats.paths
+    assert len(paths) <= num_paths
```

Comment:
since this is deterministic, can you test how many paths are present from 0 to 10 in this graph and have a specific test? thanks

Reply:
Just to confirm, do you mean calculating the number of paths in the graph and validating it? Given from what I understand there is a chance that there will be less paths than num_paths